python/thunder/utils/ec2.py

```diff
@@ -23,7 +23,6 @@
 #
```

Comment:
Was there a reason to not keep a separate executable for this one, as we're doing for the others?